slurm_ops_manager/slurm_ops_base.py

```diff
@@ -260,11 +260,69 @@ def _slurmd_group(self) -> str:
         """Return the slurmd group."""
         return "root"
 
+    @property
+    def _slurmrestd_user_id(self) -> str:
+        """Return the slurmrestd user ID."""
+        return "64031"
```

Comment:
just to confirm: does slurm 22.05.8 require this specific ID or did you just pick one that it's not intuitive to exist?